CodeStarterCamp_Week1/printPockybyorder.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+func makePockybody(topping:String, pockybody:String){
+    print(topping, terminator: "")
+    print(pockybody, separator: "", terminator: "")
+    print(topping)
+}
+
+func printPockybody(lengthofPocky:Int){
+    for _ in 1 ... lengthofPocky/2 {
```

Comment:
for loop의 조건문은 일반적으로 띄어쓰기를 하지 않습니다.(... 사이) 하지만 다른 곳은 띄어쓰기를 해주세요.
```suggestion
    for _ in 1...lengthofPocky / 2 {
```

Reply:
넵 알겠습니다